modules/apps/source-formatter-suppressions.xml

```diff
@@ -26,7 +26,6 @@
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtil\.java" />
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/java/com/liferay/dynamic/data/mapping/taglib/internal/servlet/ServletContextUtil\.java" />
 		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/java/com/liferay/dynamic/data/mapping/taglib/internal/util/PortletDisplayTemplateUtil\.java" />
-		<suppress checks="ComponentExposureCheck" files="dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/util/PortletDisplayTemplateUtil\.java" />
```

Comment:
@slnn Is the one above this also modified in this PR? Could you remove it from this file also and try SF? I think we should remove it as long as we convert the util to non-component.

Reply:
Done